bindata/network/ovn-kubernetes/self-hosted/alert-rules.yaml

```diff
@@ -328,3 +326,44 @@ spec:
       for: 10m
       labels:
         severity: critical
+    # OVN kubernetes node functional alerts
+    - alert: NodeWithoutOVNKubeNodePodRunning
+      annotations:
+        summary: All Linux nodes should be running an ovnkube-node pod, {{"{{"}} $labels.node {{"}}"}} is not.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-network-operator/NodeWithoutOVNKubeNodePodRunning.md
+        description: |
+          Networking is degraded on nodes that do not have a functioning ovnkube-node pod. Existing workloads on the
+          node may continue to have connectivity but any additional workloads will not be provisioned on the node. Any
```

Comment:
The network policy stuff seems too specific. You can just say "Networking on this node may function abnormally until the node is restored"

Reply:
Its confusing alright. I didnt mean k8 network policy, but network control plane policy.

Changed to:

```
          Networking is degraded on nodes that do not have a functioning ovnkube-node pod. Existing workloads on the
          node may continue to have connectivity but any changes to the networking control plane will not be implemented.
```